src/site/content/en/blog/streams/index.md

```diff
@@ -0,0 +1,966 @@
+---
+title: Streams—The definitive guide
+subhead: Learn how to use readable, writable, and transform streams with the Streams API.
+description: |
+  The Streams API allows JavaScript to programmatically access streams of data received over the
+  network and process them as desired.
+authors:
+  - thomassteiner
+date: 2021-01-18
+# updated: 2021-01-18
+hero: hero.jpg
+alt: A forest stream with colored fallen leaves.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - streams
+---
+
+The Streams API allows you to programmatically access streams of data received over the network and
```

Comment:
```suggestion
The Streams API allows you to programmatically access streams of data received over the network
or created by whatever means locally and
```

Reply:
I'm unsure about the wording here. It's kinda like saying "Promises allow you to programmatically access a sequence of bluetooth devices or any other local object".

While it's true that [you can get a promise for a sequence of bluetooth devices](https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-getdevices), it isn't really a direct feature of promises.

It seems more correct to say that the fetch API can be used to get a stream of network data.